napari/layers/intensity_mixin.py

```diff
@@ -106,8 +110,16 @@ def contrast_limits_range(self):
 
     @contrast_limits_range.setter
     def contrast_limits_range(self, value):
-        """Set the valid range of the contrast limits"""
+        """Set the valid range of the contrast limits.
+        If either value is "None", the current range will be preserved.
+        If the range overlaps the visible contrast limits, the range contrast
+        limits will be clipped to the range of the currently set contrast
+        limits.
+        If the requested contrast range limits are completely outside the
+        current contrast limits, the contrast limits will be overridden.
```

Comment:
Nit: override is unclear. Here's my suggestion.

```suggestion
        current contrast limits, the contrast limits will be reset to the new range.
```

Reply:
good point.